### ISSUE TITLE
Documentation isn't complete enough

### ISSUE THREAD

**Issue by pbwauyo** [aloisdeniel/media_gallery#15]
The usage documentation needs serious improvement. Its very poor and uncoordinated. Very hard to follow. Also the examples are very poor.

**Comment by aloisdeniel** [comment]
The example looks pretty complete to me, I even built an entire UI example.

What would you like in the documentation ? Maybe add a quickstart example with all functions used together ?

**Comment by pbwauyo** [comment]
for example this is a snippet from your docs:
```
final MediaPage imagePage = await widget.collection.getMedias(
    mediaType: MediaType.image,
    take: 500,
);
```
but no definition of the type of `widget.collection`

and also from another snippet:
```
MediaCollections collections = await MediaGallery.listMediaCollections(
    mediaTypes: [MediaType.image, MediaType.video],
);
```
and the type `MediaCollections`  doesnt seem to exist in the library.
Generally the examples are overly complicated and confusing.
I wish the examples were more orderly and chronological.

**Comment by aloisdeniel** [comment]
Okay thanks for the feedbacks.

**Comment by BSiddharth** [comment]
Yes really Documentation needs some improvement. After reading for 5 minutes I still don't get what does your package do? You have made an example but if I had to write all that , why am I using this. I thought this package do all that stuff for me.
Like doing something like ShowGallery( take :50, mediaType: MediaType.image,) and this will do the work which was shown in the picture

![image](https://user-images.githubusercontent.com/52542390/102015080-5b7d2080-3d7f-11eb-930f-3890e3533b25.png)

**Comment by jorjyeah** [comment]
> for example this is a snippet from your docs:
> 
> ```
> final MediaPage imagePage = await widget.collection.getMedias(
>     mediaType: MediaType.image,
>     take: 500,
> );
> ```
> 
> but no definition of the type of `widget.collection`
> 
> and also from another snippet:
> 
> ```
> MediaCollections collections = await MediaGallery.listMediaCollections(
>     mediaTypes: [MediaType.image, MediaType.video],
> );
> ```
> 
> and the type `MediaCollections` doesnt seem to exist in the library.
> Generally the examples are overly complicated and confusing.
> I wish the examples were more orderly and chronological.

So, where are the complete documentations? I'm still get confused every time I read the "example" project.
